components/sidebar/sidebar_channel/index.js

```diff
@@ -105,6 +111,8 @@ function makeMapStateToProps() {
             channelId,
             channelName: channel.name,
             channelDisplayName,
+            botIconUrl,
+            botLastIconUpdate,
```

Comment:
do we have to pass this prop down? I wonder if we can just do with a `botIconUrl: null` if bot icon is not set. Thoughts?